vms/proposervm/block/block.go

```diff
@@ -40,6 +45,20 @@ type SignedBlock interface {
 	// Proposer returns the ID of the node that proposed this block. If no node
 	// signed this block, [ids.EmptyNodeID] will be returned.
 	Proposer() ids.NodeID
+
+	VRFSig() []byte
+
+	// VerifySignature validates the correctness of the VRF signature.
+	VerifySignature(*bls.PublicKey, []byte, ids.ID, uint32) bool
```

Comment:
nit: might be good to name these parameters so we don't need to look at an implementation to see what they are.

i.e.
```
(pk *bls.PublicKey, parentVRFSig []byte, chainID ids.ID, networkID uint32)
```

Reply:
np